Tests/CredentialsTests/DummyTokenPlugin.swift

```diff
@@ -47,9 +47,9 @@ public class DummyTokenPlugin : CredentialsPluginProtocol {
                 let userProfile = UserProfile(id: "123", displayName: "Dummy User", provider: self.name)
                 let newCacheElement = BaseCacheElement(profile: userProfile)
                 #if os(OSX)
-                    self.usersCache!.setObject(newCacheElement, forKey: token as NSString)
+                    self.usersCache?.setObject(newCacheElement, forKey: token as NSString)
```

Comment:
If `self.usersCache` is nil, do we still want to call `onSuccess(userProfile)`? If yes, this looks good.

If not, maybe we should add a `guard let usersCache = self.usersCache else { ... }` at the beginning and use `usersCache` instead of `self.usersCache?`